source/Irrlicht/CGLTFMeshFileLoader.cpp

```diff
@@ -123,13 +133,7 @@ void CGLTFMeshFileLoader::loadPrimitives(
 }
 
 CGLTFMeshFileLoader::MeshExtractor::MeshExtractor(
-		const tinygltf::Model& model) noexcept
-	: m_model(model)
-{
-}
-
-CGLTFMeshFileLoader::MeshExtractor::MeshExtractor(
-		const tinygltf::Model&& model) noexcept

```

Comment:
Rule of 5. If one of the destructors, copy or move destructors is manually defined, then you probably have to define all of them.